src/FsAutoComplete/Parser.fs

```diff
@@ -170,6 +185,27 @@ module Parser =
 
       next.Invoke(ctx))
 
+  let configureOTel =
+    Invocation.InvocationMiddleware(fun ctx next ->
+
+      if ctx.ParseResult.HasOption otelTracingOption then
```

Comment:
```suggestion
      if ctx.ParseResult.GetValue otelTracingOption then
```

This ensures that if someone passes `--otel-exporter-enabled false` (which is valid out of the box with boolean options) that we listen to it correctly.